### PR TITLE
Storage swagger

### PR DESCRIPTION
**Pull request by tg-msft**
There are two commits in this PR that might be easier to review individually.

The first changes to the official versions of the Storage swagger and adds README.md transformations address any deltas.  I've regenerated which resulted in some code changes, but I think that's all generator updates.  Please review this carefully to make sure all the changes are acceptable.

The second commit updates blobs to support STG69 features.

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by rakshith91** [comment]
I would prefer merging all the changes to the preview3 branch.
https://github.com/Azure/azure-sdk-for-python/tree/storage-preview3

**Comment by tg-msft** [comment]
I was pretty sure I was missing a parameter or two.  😄   Thanks @annatisch

**Comment by annatisch** [comment]
All good - for some reason all the XML context for serialization/deserialization has been stripped out of the models and I can't figure out why....
So I my add a few more tweaks once I figure it out!

**Comment by tg-msft** [comment]
Try adding `enable-xml: true` as a setting?  Java uses that.

**Comment by annatisch** [comment]
omg - yup that was it! Thanks!

**Comment by tg-msft** [comment]
To fix the analyze step, the following "known content issues" need to be add at https://github.com/Azure/azure-sdk-for-python/blob/master/.docsettings.yml#L19

```
  - ['/sdk/storage/azure-storage-file/swagger/README.md', '#4554']
  - ['/sdk/storage/azure-storage-queue/swagger/README.md', '#4554']
  - ['/sdk/storage/azure-storage-blob/swagger/README.md', '#4554']
```

File: sdk/storage/azure-storage-blob/azure/storage/blob/_generated/models/_models_py3.py [hunk 1]
```diff
@@ -121,8 +121,8 @@ class BlobHierarchyListSegment(Model):
     }
 
     _attribute_map = {
-        'blob_prefixes': {'key': 'BlobPrefixes', 'type': '[BlobPrefix]', 'xml': {'name': 'BlobPrefixes', 'itemsName': 'BlobPrefix'}},
-        'blob_items': {'key': 'BlobItems', 'type': '[BlobItem]', 'xml': {'name': 'BlobItems', 'itemsName': 'Blob'}},
+        'blob_prefixes': {'key': 'BlobPrefixes', 'type': '[BlobPrefix]', 'xml': {'name': 'BlobPrefix', 'itemsName': 'BlobPrefix'}},
```
Comment (xiafu-msft) [inline sdk/storage/azure-storage-blob/azure/storage/blob/_generated/models/_models_py3.py]: is this manually added 😀? If so, does that mean we should edit this every time after we generate the code?

Comment (tg-msft) [reply sdk/storage/azure-storage-blob/azure/storage/blob/_generated/models/_models_py3.py]: No, it happens automatically via the transform at https://github.com/Azure/azure-sdk-for-python/blob/46f7acee340c05661ce39616b19ddcf637d0fc20/sdk/storage/azure-storage-blob/swagger/README.md#blobhierarchylistsegment

**Pull request merged**